config/shuttle.go

```diff
@@ -180,7 +181,7 @@ func NewShuttle(appVersion string) *Shuttle {
 			Queue: QueueEngine{
 				Host:      "",
 				Enabled:   false,
-				Consumers: 5,
+				Consumers: 3,
```

Comment:
Why did we change this to 3?

Reply:
given we have too many open db connections, I figured we lower these a bit